german/the-black-hack.md

```diff
@@ -46,9 +46,9 @@ Rüstung bietet Schutz, indem sie allen erlittenen Schaden reduziert. Jedes Stü
 
 |     Typ        | Punkte |
 |:--------------:|:------:|
-|  Festes Wams   |   2    |
-|  Lederrüstung  |   4    |
-|   Kettenhemd   |   6    |
+| Gambeson       |   2    |
```

Comment:
Gerade bei den Zaubern finde ich Dinge wie "Magie Entdecken", "Leichte Wunden Heilen" und "Magisches Geschoss" eigentlich extrem zugänglich. Man hätte die ja stattdessen auch "Odem Arcanum", "Balsam Salabunde" oder "Fulminictus" nennen können, was vielleicht irgendwie atmosphärisch sein kann, aber eben auch mehr Investition in das Spiel erfordert.

Für einen Begriff wie "Kettenhemd" würde mir auch keine naheliegendere Alternative einfallen. Wenn es einen gäbe, würde ich vielleicht den nehmen.

Andererseits verlasse ich mich bei der Auswahl auch auf mein persönliches Sprachgefühl. Beim *Gambeson* weiß ich, dass ich schon damals bei meiner ersten DSA-Box über das Wort gestolpert bin, dass mir das Wort noch nie außerhalb eines speziellen Rollenspielkontextes untergekommen ist, dass fast alle deutschen Treffer im Internet LARP-Seiten sind und dass Duden.de das Wort ausschließlich als Übersetzung aus dem Englischen kennt. Das erweckt bei mir halt den Eindruck, dass das eher so ein "Flim Flam Funkel"-Begriff ist, den ich vielleicht besser mit "Licht" übersetzen würde...